compiler/src/test/kotlin/symboltest.kt

```diff
@@ -63,7 +65,7 @@ class SymbolTest {
 
     /**
      * Returns the list of reserved words for testing data. Note that this cannot be passed to test by:
-     * @ValueSource("strings = RESERVED_WORDS") as JUnit5 complains that the source is not compile-time static
+     * @ValueSource("strings = RESERVED_WORDS") as JUnit5 complains that the source is not compileSource-time static
```

Comment:
@ValueSource is the first symbol in the javadoc, so the line dissappears, which is not what you want. Also "compile-time" should not have been renamed to "compileSource-time". Be careful with comments when refractoring names with the rename function.

Reply:
Yes, the rename-function can be a bit greedy. I never noticed that the actual rendering of the javadoc ate the annotation - added single-quotes to it in 6a07091.